### PR TITLE
initiate ci/cd

### PR DESCRIPTION
**Pull request by ajib-jawwad**

### CODE REVIEW

**Review by infra-gl — COMMENTED** [review]
<details>
  <summary>Trivy Result</summary>

  ```
2023-12-05T07:39:22.381Z	[33mWARN[0m	'--light' option is deprecated and will be removed. See also: https://github.com/aquasecurity/trivy/discussions/1649
2023-12-05T07:39:22.401Z	[34mINFO[0m	Need to update DB
2023-12-05T07:39:22.401Z	[34mINFO[0m	DB Repository: ghcr.io/aquasecurity/trivy-db
2023-12-05T07:39:22.401Z	[34mINFO[0m	Downloading DB...
2023-12-05T07:39:24.846Z	[34mINFO[0m	Vulnerability scanning is enabled
2023-12-05T07:39:24.846Z	[34mINFO[0m	Secret scanning is enabled
2023-12-05T07:39:24.846Z	[34mINFO[0m	If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2023-12-05T07:39:24.846Z	[34mINFO[0m	Please see also https://aquasecurity.github.io/trivy/v0.36/docs/secret/scanning/#recommendation for faster secret detection
2023-12-05T07:39:37.722Z	[34mINFO[0m	Detected OS: alpine
2023-12-05T07:39:37.723Z	[34mINFO[0m	This OS version is not on the EOL list: alpine 3.18
2023-12-05T07:39:37.723Z	[34mINFO[0m	Detecting Alpine vulnerabilities...
2023-12-05T07:39:37.735Z	[34mINFO[0m	Number of language-specific files: 1
2023-12-05T07:39:37.735Z	[34mINFO[0m	Detecting node-pkg vulnerabilities...

170848542527.dkr.ecr.ap-southeast-3.amazonaws.com/glair-genai-doc-external:latest (alpine 3.18.5)
=================================================================================================
Total: 0 (MEDIUM: 0, HIGH: 0, CRITICAL: 0)

2023-12-05T07:39:37.778Z	[34mINFO[0m	Table result includes only package filenames. Use '--format json' option to get the full path to the package file.

Node.js (node-pkg)
==================
Total: 2 (MEDIUM: 1, HIGH: 1, CRITICAL: 0)

┌────────────────────────┬─────────────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────┐
│        Library         │    Vulnerability    │ Severity │ Installed Version │ Fixed Version │                          Title                          │
├────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ postcss (package.json) │ CVE-2023-44270      │ MEDIUM   │ 8.4.14            │ 8.4.31        │ An issue was discovered in PostCSS before 8.4.31. The   │
│                        │                     │          │                   │               │ vulnerability af ......                                 │
│                        │                     │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-44270              │
├────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ sharp (package.json)   │ GHSA-54xq-cgqr-rpm3 │ HIGH     │ 0.32.4            │ 0.32.6        │ sharp vulnerability in libwebp dependency CVE-2023-4863 │
│                        │                     │          │                   │               │ https://github.com/advisories/GHSA-54xq-cgqr-rpm3       │
└────────────────────────┴─────────────────────┴──────────┴───────────────────┴───────────────┴─────────────────────────────────────────────────────────┘
  ```
</details>

File: doc/scripts/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
+
+
+USER nextjs
+
+EXPOSE 3000
+
+ENV PORT 3000
+
+CMD ["node", "server.js"]
```
Comment (timotiusnc) [inline doc/scripts/docker/Dockerfile:51]: newline pls. Also on other files.

**Review by infra-gl — COMMENTED** [review]
<details>
  <summary>Trivy Result</summary>

  ```
2023-12-05T09:45:20.104Z	[33mWARN[0m	'--light' option is deprecated and will be removed. See also: https://github.com/aquasecurity/trivy/discussions/1649
2023-12-05T09:45:20.117Z	[34mINFO[0m	Need to update DB
2023-12-05T09:45:20.117Z	[34mINFO[0m	DB Repository: ghcr.io/aquasecurity/trivy-db
2023-12-05T09:45:20.117Z	[34mINFO[0m	Downloading DB...
2023-12-05T09:45:23.492Z	[34mINFO[0m	Vulnerability scanning is enabled
2023-12-05T09:45:23.492Z	[34mINFO[0m	Secret scanning is enabled
2023-12-05T09:45:23.492Z	[34mINFO[0m	If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2023-12-05T09:45:23.492Z	[34mINFO[0m	Please see also https://aquasecurity.github.io/trivy/v0.36/docs/secret/scanning/#recommendation for faster secret detection
2023-12-05T09:45:38.431Z	[34mINFO[0m	Detected OS: alpine
2023-12-05T09:45:38.431Z	[34mINFO[0m	This OS version is not on the EOL list: alpine 3.18
2023-12-05T09:45:38.432Z	[34mINFO[0m	Detecting Alpine vulnerabilities...
2023-12-05T09:45:38.434Z	[34mINFO[0m	Number of language-specific files: 1
2023-12-05T09:45:38.434Z	[34mINFO[0m	Detecting node-pkg vulnerabilities...

170848542527.dkr.ecr.ap-southeast-3.amazonaws.com/glair-genai-doc-external:latest (alpine 3.18.5)
=================================================================================================
Total: 0 (MEDIUM: 0, HIGH: 0, CRITICAL: 0)

2023-12-05T09:45:38.522Z	[34mINFO[0m	Table result includes only package filenames. Use '--format json' option to get the full path to the package file.

Node.js (node-pkg)
==================
Total: 2 (MEDIUM: 1, HIGH: 1, CRITICAL: 0)

┌────────────────────────┬─────────────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────┐
│        Library         │    Vulnerability    │ Severity │ Installed Version │ Fixed Version │                          Title                          │
├────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ postcss (package.json) │ CVE-2023-44270      │ MEDIUM   │ 8.4.14            │ 8.4.31        │ An issue was discovered in PostCSS before 8.4.31. The   │
│                        │                     │          │                   │               │ vulnerability af ......                                 │
│                        │                     │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-44270              │
├────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ sharp (package.json)   │ GHSA-54xq-cgqr-rpm3 │ HIGH     │ 0.32.4            │ 0.32.6        │ sharp vulnerability in libwebp dependency CVE-2023-4863 │
│                        │                     │          │                   │               │ https://github.com/advisories/GHSA-54xq-cgqr-rpm3       │
└────────────────────────┴─────────────────────┴──────────┴───────────────────┴───────────────┴─────────────────────────────────────────────────────────┘
  ```
</details>

**Pull request merged**